modules/distr-tracing-config-storage.adoc

```diff
@@ -615,3 +649,90 @@ spec:
 <4> Volume mounts and volumes which are mounted into all storage components.
+
+[id="distr-tracing-manage-es-certificates_{context}"]
+= Managing certificates with Elasticsearch
+
+You can create and manage certificates using the Red Hat Elasticsearch Operator. Managing certificates using the Red Hat Elasticsearch Operator also lets you use a single Elasticsearch cluster with multiple Jaeger Collectors.
+
+[IMPORTANT]
+====
+Managing certificates with Elasticsearch is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production.
+
+These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process. For more information about the support scope of Red Hat Technology Preview features, see https://access.redhat.com/support/offerings/techpreview/.
+====
+
+Starting with version 2.4, the {JaegerName} Operator delegates certificate creation to the Red Hat Elasticsearch Operator by using the following annotations in the Elasticsearch custom resource:
+
+* `logging.openshift.io/elasticsearch-cert-management: true`
+* `logging.openshift.io/elasticsearch-cert.jaeger-shared-es: user.jaeger`
+* `logging.openshift.io/elasticsearch-cert.curator-shared-es: system.logging.curator`
+
+.Example Elasticsearch CR showing annotations
+[source,yaml]
+----
+apiVersion: logging.openshift.io/v1
+kind: Elasticsearch
+metadata:
+  annotations:
+    logging.openshift.io/elasticsearch-cert-management: true
+    logging.openshift.io/elasticsearch-cert.jaeger-shared-es: user.jaeger
+    logging.openshift.io/elasticsearch-cert.curator-shared-es: system.logging.curator
+  name: elasticsearch
+spec:
+  managementState: Managed
+  nodeSpec:
+    resources:
+      limits:
+        memory: 16Gi
+      requests:
+        cpu: 1
+        memory: 16Gi
+  nodes:
+    - nodeCount: 3
+      proxyResources: {}
+      resources: {}
+      roles:
+        - master
+        - client
+        - data
+      storage: {}
+  redundancyPolicy: ZeroRedundancy
+----
+
+.Prerequisites
+
+* {product-title} 4.7
+* {logging-title} 5.2
+
+You enable certificate management by setting `spec.storage.elasticsearch.useCertManagement` to `true` in the Jaeger custom resource.
+
+.Example showing `useCertManagement`
+[source,yaml]
+----
+apiVersion: jaegertracing.io/v1
+kind: Jaeger
+metadata:
+  name: jaegerprod
+  namespace: tracing-system
+spec:
+  strategy: production
+  storage:
+    type: elasticsearch
+    elasticsearch:
+      name: elasticsearch
+      doNotProvision: true
+      useCertManagment: true
+      nodeCount: 3
+      resources:
+        requests:
+          cpu: 1
+          memory: 16Gi
+        limits:
+          memory: 16Gi
+      useCertManagment: true
+----
```

Comment:
@JStickler remove `nodeCount` and `resources` is not taken care by `jaeger`, as `doNotProvision: true`. also remove the `namespace`.

Reply:
@jkandasa Please check and make sure I correctly interpreted which lines to edit.

Reply:
good catch @jkandasa! As far as i can see, it looks fine :+1:

Reply:
@JStickler typo in 
```
      useCertManagment: true
```
Other than this all other changes looks good to me.